pyproject.toml

```diff
@@ -51,6 +51,8 @@ doc = [
 check = [
 	"pytest-checkdocs >= 2.4",
 	"pytest-ruff >= 0.2.1; sys_platform != 'cygwin'",
+	# Deprecated rules must now be selected by exact rule code
+	"ruff >= 0.13; sys_platform != 'cygwin'",
```

Comment:
Alternatively, this can be set directly in `ruff.toml` with [required-version](https://docs.astral.sh/ruff/settings/#required-version). The UX is not great as it fails at runtime telling you to bust your cache, also https://github.com/astral-sh/ruff/issues/19922